app/models/pusher.rb

```diff
@@ -110,7 +110,7 @@ def update_remote_bundler_api(to = RestClient)
     }.to_json
 
     begin
-      timeout(5) do
+      ::Timeout.timeout(5) do
```

Comment:
can we set a timeout on the `RestClient` instance instead? 
As far as I remember ruby `timeout` is racy for some reason, so if we could have an timeout on the http connection would be better.